assignments/callbacks.js

```diff
@@ -67,8 +69,23 @@ function find(b, arr) {
 console.log(contains("Pencil", items, find));
 /* STRETCH PROBLEM */
 
+let x = [1, 2, 3, 4, 5, 6, 1, 2, 4, 5];
 function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+  return cb(y);
 }
+let y = [];
+function dupli() {
+  x.forEach(ele => {
+    if (y.indexOf(ele) === -1) {
+      y.push(ele);
+    }
+  });
+}
```

Comment:
looks good, maybe try and find ways to do this with other array methods like .map